src/tools/fuse/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+license_files = LICENSE.md
```

Comment:
Can you specify the top-level LICENSE.md here?

Reply:
Yes, I changed it to use the top-level license in cc580244ec501d2e8c18b4e4f45e216bb7fd9155.